src/tests/test_utils/common_test_utils/include/common_test_utils/ov_test_utils.hpp

```diff
@@ -71,3 +71,16 @@ std::shared_ptr<ov::op::v0::Constant> create_constant(const std::vector<T>& data
 }
 
 std::shared_ptr<ov::op::v0::Constant> create_zero_constant(const ov::element::Type_t& et, const ov::Shape& shape);
+
```

Comment:
Please, change the file name. NAme is a bit confusing

Reply:
Yes, in general we need to edit all file names starting with `ov_`